Makefile

```diff
@@ -1 +1,14 @@
-# TODO: redo this makefile properly
+EXECUTABLE:=_build/default/driver/mcltt.exe
+
+.PHONY: all clean
+
+all:
+	@cd theories; make
+	@dune build
+
+clean:
+	@cd theories; make clean
+	@dune clean
+	@echo "Cleaning finished."
+
+${EXECUTABLE}: all
```

Comment:
Do we need this target? It seems up-side-down in the sense that `$(EXECUTABLE)` is a part of "build all".

Reply:
yeah probably not